### ISSUE TITLE
run with scaled observations 

### ISSUE THREAD

**Issue by wdwatkins** [USGS-R/lake-temperature-neural-networks#3]
I have currently only scaled the input features, probably worth applying the same transformation to the observations also

**Comment by aappling-usgs** [comment]
I thought one didn't usually scale the outputs for regression problems?

**Comment by wdwatkins** [comment]
Yeah, I remember reading that somewhere too.  Maybe worth a shot though?
